lib/Client.js

```diff
@@ -623,22 +626,42 @@ class Client extends EventEmitter {
     * @arg {String} [options.avatarURL] A URL for a custom avatar, defaults to webhook default avatar if not specified
     * @arg {Boolean} [options.tts=false] Whether the message should be a TTS message or not
     * @arg {Boolean} [options.wait=false] Whether to wait for the server to confirm the message create or not
-    * @arg {Boolean} [options.disableEveryone] Whether to filter @everyone/@here or not (overrides default)
+    * @arg {Object} [options.allowedMentions] A list of mentions to allow (overrides default)
+    * @arg {Boolean} [options.allowedMentions.everyone] Whether or not to allow @everyone/@here.
+    * @arg {Boolean | Array<String>} [options.allowedMentions.roles] Whether or not to allow all role mentions, or an array of specific role mentions to allow.
+    * @arg {Boolean | Array<String>} [options.allowedMentions.users] Whether or not to allow all user mentions, or an array of specific user mentions to allow.
     * @returns {Promise<Message?>}
     */
     executeWebhook(webhookID, token, options) {
         if(!options.content && !options.file && !options.embeds) {
             return Promise.reject(new Error("No content, file, or embeds"));
         }
-        if(options.content && (options.disableEveryone !== undefined ? options.disableEveryone : this.options.disableEveryone)) {
-            options.content = options.content.replace(/@everyone/g, "@\u200beveryone").replace(/@here/g, "@\u200bhere");
+        if(options.allowedMentions || this.options.allowedMentions) {
+            const allowed = options.allowedMentions ? options.allowedMentions : this.options.allowedMentions;
```

Comment:
Move this line outside the if statement and then the if can be as simple as `if (allowed)` making it slightly less code.

Reply:
Also i don't think you need a ternary here. It could just be `alloweMentions || xxxx`